app/serializers/user_serializer.rb

```diff
@@ -3,9 +3,13 @@
 class UserSerializer < ApplicationSerializer
   include Avatarable
 
-  attributes :id, :name, :email, :provider, :language, :avatar
+  attributes :id, :name, :email, :provider, :language, :avatar, :created_at
```

Comment:
Just add this - there already is a `created_at` method in `ApplicationSerializer`

Reply:
Yeah but it displays the time at the end, but in this case I don't want to display the time, so they are a bit different